jobbergate-slurp/slurp/main.py

```diff
@@ -55,6 +56,7 @@ def migrate(
     """
     Migrates data from the legacy database to the nextgen database.
     """
+    logger.add(f"file_{datetime.now().replace(microsecond=0).isoformat()}.log")
```

Comment:
Please use `.format()` or pre-compute the string if you are going to use f-strings. Having this much logic inside an f-string gets messy and makes it harder to read the strings.

Reply:
solved in https://github.com/omnivector-solutions/jobbergate/pull/181